tool-populate/index.js

```diff
@@ -104,24 +104,44 @@ function destroyObjects(objects) {
   }
 }
 
+function setRoleACLs() {
```

Comment:
You're right, the admins should be able to modify roles, however we shouldn't set publicRead false.  I do think publicWrite false is redundant, but doesn't hurt.